mlir/test/Target/LLVMIR/Import/basic.ll

```diff
@@ -87,6 +87,38 @@ define i32 @invokeLandingpad() personality ptr @__gxx_personality_v0 {
   ret i32 0
 }
 
+declare i32 @foo2()
+
+; CHECK-LABEL: @invokePhi
+; CHECK-SAME:            (%[[cond:.*]]: i1) -> i32
+define i32 @invokePhi(i1 %cond) personality ptr @__gxx_personality_v0 {
+entry:
+  ; CHECK: %0 = llvm.mlir.constant(0 : i32) : i32
+  ; CHECK: llvm.cond_br %arg0, ^bb1, ^bb2
+  br i1 %cond, label %call, label %nocall
+; CHECK: ^bb1:  // pred: ^bb0
+call:
+  ; CHECK: %1 = llvm.invoke @foo2() to ^bb3 unwind ^bb5 : () -> i32
+  %invoke = invoke i32 @foo2() to label %bb0 unwind label %bb1
+; CHECK: ^bb2:  // pred: ^bb0
+nocall:
+  ; CHECK: llvm.br ^bb4(%0 : i32)
+  br label %bb0
+; CHECK: ^bb3:  // pred: ^bb1
+  ; CHECK: llvm.br ^bb4(%1 : i32)
+; CHECK: ^bb4(%2: i32):  // 2 preds: ^bb2, ^bb3
+bb0:
+  %ret = phi i32 [ 0, %nocall ], [ %invoke, %call ]
```

Comment:
can you test for multiple phis ? something like that (also test operand unrelated to invoke)

```suggestion
  %ret = phi i32 [ 0, %nocall ], [ %invoke, %call ]
  %foo = phi i32 [ 0, %nocall ], [ 1, %call ]
```